src/filters/Filter.ts

```diff
@@ -69,10 +69,11 @@ export interface FilterOptions
     blendMode?: BLEND_MODES;
     /**
      * the resolution the filter should be rendered at. The lower the resolution, the more performant
-     * the filter will be, but the lower the quality of the output. (defaults to the renderers resolution)
```

Comment:
Incorrect doc. Default is 1. (Though `null` as default would be nicer, I think).